terraform/xmtp-validation-service/_variables.tf

```diff
@@ -0,0 +1,54 @@
+variable "vpc_id" {
+  description = "The VPC ID"
+}
+
+variable "cluster_id" {
+  description = "The ID of the cluster to deploy into"
+}
+
+variable "private_subnets" {
+  description = "The (private) subnets used to run the service"
+}
+
+variable "cpu" {
+  description = "Available CPU for the node container"
+  default     = 2048
+}
+
+variable "memory" {
+  description = "Available memory for the node container"
+  default     = 4096
+}
+
+variable "env" {
+  description = "Environment"
+  default     = "testnet"
+}
+
+variable "docker_image" {
+  description = "Docker image for service"
+  default     = "ghcr.io/xmtp/mls-validation-service:latest"
+}
+
+variable "service_discovery_namespace_name" {
+  description = "The name of the service discovery namespace"
+}
+
+variable "allowed_ingress_cidr_blocks" {
+  description = "The CIDR blocks allowed to connect to the service"
+  type        = list(string)
+}
+
+variable "chain_rpc_urls" {
```

Comment:
are these used by the validation service? I am not certain...

Reply:
I believe they are used by the `chain_rpc_verifier` in `libxmtp` (in `xmtp_id/scw_verifier`, which is referenced by `mls_validation_service`)